ajax/libs/g2plot/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "g2plot",
+  "filename": "dist/g2plot.js",
```

```suggestion
  "filename": "g2plot.js",
```
As Sven mentioned, this should not include `dist`, as the auto-update config will pull assets out of this directory into the root of the version.